### ISSUE TITLE
get_extension_funcs () is not supported

### ISSUE THREAD

**Issue by alex-mashin** [facebook/hhvm#6497]
Under HHVM 3.10 and 3.10.1 and probably under several previous versions an attemp to call `get_extension_funcs ('(extension)')` causes a fatal error:
```
Fatal error: f_get_extension_funcs is not supported: extensions are built differently in (path to file).php on line ...
```

This breaks at least one extension: luasandbox (see https://phabricator.wikimedia.org/T115779).

**Comment by SiebelsTim** [comment]
This is present for like always. You can track it down to HHVM 2.x. So this is not about an updated hhvm version, it's about code that changed in luasandbox I assume.

@sgolemon attached a TODO to the function https://github.com/facebook/hhvm/blob/a9942783805f0840fdc35fce60598309ad8b8b57/hphp/runtime/ext/std/ext_std_options.cpp#L246

**Comment by alex-mashin** [comment]
So, there is a way to build luasanbox under HHVM 3.11 (and earlier versions, I suppose): just add some parametres to `cmake .` (see https://phabricator.wikimedia.org/T115779#1892540).

Whether this remains a bug than needs fixing, since there is still an incompability with PHP, I don't know.

**Comment by Orvid** [comment]
This should be fairly easy to implement, just capture the functions as they get registered.

**Comment by Orvid** [comment]
Implemented in 3925e0c2ccab87656f3d687e71632218f7d25d1d.
